.changeset/nine-ads-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': minor
+---
+
+feat: add support for Rolldown
```

Comment:
it would be nice to expand this to tell people how to setup the override and install the required peer deps so that we have something to point to if they ask how to test out rolldown